plugins/modules/vmware_guest.py

```diff
@@ -242,6 +242,22 @@
         iommu:
             type: bool
             description: Flag to specify if I/O MMU is enabled for this virtual machine.
+  encryption:
+    type: dict
+    default: {}
+    description:
+    - Manage virtual machine encryption settings
+    - All parameters case sensitive.
+    version_added: '3.8.0'
```

Comment:
```suggestion
    version_added: '3.9.0'
```

I'm afraid I've just released 3.8.0 and due to the CI problems this PR didn't make it into this release. Let's hope we'll get it into 3.9.0.